ts_src/psetv2/input.ts

```diff
@@ -261,7 +261,7 @@ export class PsetInput {
           input.tapLeafScript!.push({
             controlBlock,
             leafVersion,
-            script: kp.value,
+            script: kp.value.slice(0, kp.value.length - 1),
```

Comment:
should we check the size of the bytes passed before slicing?

Reply:
hmmm not sure what should be the check as the scipt has a variable size

Reply:
note that [].slice(0, -1) does not panic